CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Secure worker connections are no longer supported for Editor builds. They are still supported for packaged builds.
 
 ### Features:
+- New default Control GameMode in the Example Project. Two Teams compete to control points on the map, with help from NPCs guards.
```

Comment:
```suggestion
- New default Control GameMode in the Example Project. Two Teams compete to control points on the map, with help from NPC guards.
```
Nit.
Also, maybe it should be even more prominent? Not sure if we have a section for this, but big shiny new features should have a section, I think.

Reply:
Hmm, new section maybe up to @ernopp / @oblm ?